crates/llvm-builder/tests/build.rs

```diff
@@ -53,6 +53,16 @@ fn clone_build_and_clean_musl() -> anyhow::Result<()> {
         .assert()
         .success();
 
+    Command::cargo_bin(common::REVIVE_LLVM)?
+        .current_dir(test_dir.path())
+        .arg("build")
+        .arg("--llvm-projects")
+        .arg("clang")
+        .arg("--llvm-projects")
+        .arg("lld")
+        .assert()
+        .success();
```

Comment:
Marking it here but can you remove the `builtins` command below as a drive-by? The builtins are built anyways during the `build` command.